tests/e2e/app-specs/verify-extensions.e2e-spec.ts

```diff
@@ -56,6 +62,25 @@ const ENTITY_MAP = new Map<string, any>()
         title: dynamicItemMock.data.Title
     });
 
+describe("Verify protected route", () => {
+    it("verify config guard", async () => {
+        await BrowserNavigate(PRINT_PREVIEW_URL);
+        const currentUrl = await BrowserGetUrl();
+        expect(currentUrl).toBe(CONFIG_PAGE_URL);
+    });
+
+    it("verify auth guard", async () => {
+        await BrowserNavigate(BASE_URL);
+        const setLocalStorageString = `localStorage.setItem("sf.config.serviceUrl","${URL_IN_CONFIG_FILE}");`;
+        await browser.executeScript(setLocalStorageString);
```

Comment:
we have a browser-helper.ts which holds many 1 line functions that execute browser functions. Maybe export a function from there and use it here just for consistency?

Reply:
done